File: java/client/src/main/java/glide/api/models/exceptions/ConnectionException.java
/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
package glide.api.models.exceptions;

/**
 * Connection error: Errors that are thrown when a connection disconnects. These errors can be
 * temporary, as the client will attempt to reconnect.
 */
public class ConnectionException extends GlideException {
    public ConnectionException(String message) {
        super(message);
    }
}

File: csharp/sources/Valkey.Glide.InterOp/Errors/ConnectionException.cs
namespace Valkey.Glide.InterOp.Errors;

/// <summary>
/// A connection problem between Glide and Valkey.<br />
/// That error is thrown when a connection disconnects. These errors can be temporary, as the client will attempt to reconnect.
/// </summary>
public sealed class ConnectionException : GlideException
{
    public ConnectionException() { }

    public ConnectionException(string message) : base(message) { }

    public ConnectionException(string message, Exception innerException) : base(message, innerException) { }
}